lib/ui/painting/image_generator_registry_unittests.cc

```diff
@@ -0,0 +1,104 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/lib/ui/painting/image_generator_registry.h"
+
+#include "flutter/fml/mapping.h"
+#include "flutter/shell/common/shell_test.h"
+#include "flutter/testing/testing.h"
+
+namespace flutter {
+namespace testing {
+
+static sk_sp<SkData> LoadValidImageFixture() {
+  // Load the fixture with fml mappings.
+  auto fixtures_directory =
+      fml::OpenDirectory(GetFixturesPath(), false, fml::FilePermission::kRead);
+  auto fixture_mapping = fml::FileMapping::CreateReadOnly(
```

Comment:
This works too but `OpenFixtureAsMapping` does exactly the same thing. Also, error checking in case the fixture cannot be opened.

Reply:
Oh nice, I somehow missed that utility. Done.